sideboard/lib/sa/__init__.py

```diff
@@ -261,7 +282,7 @@ def resolve_model(cls, name):
                     return subclasses[singular]
 
             if name.lower().endswith('ies'):
-                singular = name[:-3] + 'sy'  # TODO: sy looks like a typo, and we need to either make this better or get rid of it
+                singular = name[:-3] + 'y'
```

Comment:
Funny enough, PyCharm actually warns you about TODOs you left in the code when you try to commit code. Clearly, they agree!